available-connectors.json

```diff
@@ -69,13 +70,14 @@
     "description": "IBM DB2",
     "baseModel": "PersistedModel",
     "features": {
-      "discovery": false,
+      "discovery": true,
       "migration": false
     },
     "settings": {
       "dsn": {
         "type": "string",
-        "description": "Connection String dsn to override other settings (eg: DATABASE=MY_DB;HOSTNAME=MY_HOST;PORT=MY_PORT;PROTOCOL=TCPIP;UID=MY_UID;PWD=MY_PWD))"
+        "description":
```

Comment:
Unrelated white-space change, please revert here and in all other places below too.